src/entities/get-offerings-params.ts

```diff
@@ -8,4 +18,10 @@ export interface GetOfferingsParams {
    * If not specified, the default currency will be used.
    */
   readonly currency?: string;
+
+  /**
+   * The identifier of the offering to fetch.
+   * Can be a string identifier or one of the predefined keywords.
+   */
+  readonly offeringIdentifier?: string | OfferingKeyword;
```

Comment:
One thing to bear in mind is that `GetOfferingsParams` is also used in [getCurrentOfferingForPlacement](https://github.com/RevenueCat/purchases-js/pull/201/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dL242), but it's completely ignore in that function. Not sure how do you feel about it.

Reply:
Added a suggestion on how to handle it.